### ISSUE TITLE
Create a Demo video

### ISSUE THREAD

**Issue by mmccool** [w3c/wot-marketing#80]
For TPAC, we have an opportunity to host a breakout session which can be used to introduce new people to WoT.  In the past during F2F TPAC meetings we have set up physical demos.   We could ALSO do online demos during this TPAC but the logistics would be difficult.   So I propose we edit together a short demo video, about 15m long, using contributions from various people, with maybe a few bits from the plugfest, and then (for the breakout) add 45m of Q&A.   The video can then be used as marketing collateral.  This could be videos of actual systems or slides (eg in case of software like a directory service) but in any case should have a voice-over explanation.
To make things more consistent, we can:
- Limit each contributions to 3m
- Need a introduction and conclusion segments (may be able to reuse material from 3m "WoT introduction" video being done for Ralph)
- Can be "individual" system video segments first, followed by "interop" segments
- Maybe have a single narrator
- Decide on some speaking points, use cases, common scenarios
- Guidelines for video style, e.g. slides should use the WoT template

**Comment by mmccool** [comment]
Update:
- Actually two calls now, one for a 3m "group introduction" and another for demo videos.
- Sept 21 deadline is firm...
- Intro video can just be a voice-over on some powerpoint slides, captured with Zoom (we discussed other tools as well)
- I'm working on the intro video (unfortunately not done yet; will get it done today so we can look at it before the deadline...)
- As for demo videos, we have been collecting links to existing videos and putting them in a "videos" directory.  We recently discussed how to manage and distribute such content; the W3C YouTube channel has been mentioned
- We should use a common template if possible, with consistent opening and closing frames.   I'm proposing we use the PPT template for that (so videos made using PPT will be easy to match to the template, and other content can be edited with openers and closers created from PPT, and so videos are consistent with presentations...)
